sql/core/src/test/resources/sql-tests/results/operators.sql.out

```diff
@@ -357,87 +357,87 @@ struct<COT(-1):double>
 -- !query
 select ceiling(0)
 -- !query schema
-struct<ceiling(0):bigint>
+struct<ceiling(0, 0):int>
```

Comment:
Not every DBMS system supports the second parameter. Some of the external connectors are relying on the SQL representation string of expressions.
So, shall we hide the 2nd parameter if it is the default 0?

Reply:
I'm not sure how to do it. Could you please show me an example in the code ? thanks.

Reply:
You can follow `Rand`, which needs to hide the seed if not specified.

Reply:
ok, thanks !